pkg/proxy/iptables/proxier.go

```diff
@@ -868,6 +868,12 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 	}
 
+	// ensure KUBE-MARK-DROP chain exist but not do any change
+	if _, err := proxier.iptables.EnsureChain(utiliptables.TableNAT, KubeMarkDropChain); err != nil {
```

Comment:
Similar to iptablesJumpChains can we have a variable like `iptablesEnsureChain` containing only KubeMarkDropChain?

Reply:
That will be more readable. Addressed : )